container_workflow_tool/main.py

```diff
@@ -211,18 +219,20 @@ def _get_images(self):
                 images += i
         return self._filter_images(images)
 
-    def _filter_images(self, base):
+    def _filter_images(self, base: List) -> List:
         if self.do_image:
             return [i for i in base if i["component"] in self.do_image]
         elif self.exclude_image:
             return [i for i in base if i["component"] not in self.exclude_image]
         else:
             return base
 
-    def _prebuild_check(self, image_set, branches=[]):
+    def _prebuild_check(self, image_set: List, branches: List = None):
         tmp = self._get_tmp_workdir(setup_dir=False)
         if not tmp:
-            raise RebuilderError("Temporary directory structure does not exist. Pull upstream first.")
+            raise RebuilderError(
+                "Temporary directory structure does not exist. Pull upstream first."
+            )
```

Comment:
I would rather use the previous version here as I do not really like the hanging indent style used. If needed you can use a new variable (eg. `msg`) to assign the output into if line length is the issue.

Same for other instances further down the diff.